.github/workflows/publish-npm-rc.yaml

```diff
@@ -18,28 +20,146 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v3
         with:
-          node-version: '14'
+          node-version: '16'
+
+      - name: 🏷 Check tags
+        run: |
+          CURRENT_TAG=$(git describe --abbrev=0)
+
+          # `github.ref_name` is something like `release/minor/v1.1.0`.
+          # `basename` gets us `v1.1.0`, then parameter expansion strips the `v`.
+          # See https://stackoverflow.com/questions/6594085/remove-first-character-of-a-string-in-bash.
+          BRANCH_TAG=$(basename ${{ github.ref_name }})
+          BRANCH_TAG=${BRANCH_TAG:1}
+
+          echo
+          echo "Current tag is $CURRENT_TAG"
+          echo "Branch tag is $BRANCH_TAG"
+          echo "Comparing tags with \"yarn dlx semver-compare-cli $CURRENT_TAG lt $BRANCH_TAG\""
+          echo
+
+          # See if the current tag is less than the branch's tag.
+          # If it is, we're good to keep going.
+          yarn dlx semver-compare-cli $CURRENT_TAG lt $BRANCH_TAG
```

Comment:
All this looks good and I want to merge to take it for a test drive. The only question I have is this line — when I run it locally it doesn't exit anything. Have you been able to confirm this will pass/fail the Job?

Reply:
Yeah it'll pass/fail the job based on its exit code. Not the friendliest thing; I should probably just make this a JS action for better logging, but started with CLI stuff cause it seemed so simple. But yeah, if the current tag isn't less than the branch tag, this'll exit 1, failing the whole action.